recipes/_configure.rb

```diff
@@ -58,8 +59,8 @@
   action :create
 end
 
+# set env variables for runit to use
 (sentry_config["additional_env_vars"] || {}).each do |key, value|
```

Comment:
You shouldn't need the `|| {}` if the default for `additional_env_vars` is a hash/mash